mongorestore/restore.go

```diff
@@ -25,6 +25,33 @@ import (
 
 const insertBufferFactor = 16
 
+var indexOptions = map[string]bool{
+	"bits": true,
+	"min":  true,
+	"max":  true,
+	"coarsestIndexedLevel": true,
```

Comment:
This (and finest) aren't in the documented index options.  Can you find out if they would actually appear in an index definition document?

Reply:
It's undocumented but they can appear. I found an old JIRA ticket which suggests Lyft uses them. I'll open a DOCS ticket about it.